client/landing/stepper/declarative-flow/site-setup-flow.ts

```diff
@@ -240,6 +240,8 @@ const siteSetupFlow: Flow = {
 					if ( selectedDesign?.design_type === 'assembler' ) {
 						window.sessionStorage.setItem( 'wpcom_signup_completed_flow', 'pattern_assembler' );
 						return exitFlow( `/site-editor/${ siteSlug }` );
+					} else if ( isBlankCanvasDesign( selectedDesign ) ) {
```

Comment:
@autumnfjeld Do we want to show the first slide of welcome tour from the assembler to the user select the BCPA CTA and land on the editor directly as well?

Reply:
I just tested this case, I think it makes sense to show the Welcome tour for the users that click "Open the editor". I'll update the PR. Thanks.